File: accidentes/FicheroAccidentes-Java-IvanRC/src/main/java/model/Accidente.java
package model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.time.LocalTime;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Accidente {
    String numeroExpediente; // 0
    LocalDate fecha; // 1
    LocalTime hora; // 2
    List<String> localizacion; // 3
    Integer numeroCalle; // 4, en caso de que no sea válido pues un nulo
    Distrito distrito;
    String tipoAccidente; // 7
    String estadoMeteorologico; // 8
    String tipoDeVehiculo; // 9
    String tipoDePersona; // 10
    String rangoEdad; // 11
    String sexo; // 12
    Lesividad lesividad;
    Double longitud; // 15
    Double altitud; // 16
    Boolean esPositivaEnAlchol; // 17, N == false y S == true
    Boolean esPositivaEnDrogas; // 18, NULL == false y 1 == true
}

File: accidentes/FicheroAccidentes-Kotlin-IvanRC/src/main/kotlin/model/Accidente.kt
package model

import java.time.LocalDate
import java.time.LocalTime

data class Accidente (
    val numeroExpediente: String, // 0
    val fecha: LocalDate, // 1
    val hora: LocalTime, // 2
    val localizacion: List<String>, // 3
    val numeroCalle: Int?, // 4, en caso de que no sea válido pues un nulo
    val distrito: Distrito,
    val tipoAccidente: String, // 7
    val estadoMeteorologico: String, // 8
    val tipoDeVehiculo: String, // 9
    val tipoDePersona: String, // 10
    val rangoEdad: String, // 11
    val sexo: String, // 12
    val lesividad: Lesividad,
    val longitud: Double?, // 15
    val altitud: Double?, // 16
    val esPositivaEnAlchol: Boolean, // 17, N == false y S == true
    val esPositivaEnDrogas: Boolean // 18, NULL == false y 1 == true
)